src/components/TabletApp/TabletApp.tsx

```diff
@@ -10,7 +10,7 @@ import Menu from "./RightSide/Menu/Menu";
 import CONSTANTS from "../../constants/Constants";
 import { getMeetings, getMeetingsData } from "../../api/getRequests";
 import COLORS from "../../constants/CustomColors";
-import getRoomStatus from "../../functions/getRoomStatus";
+import getRoomStatus from "../../functions/GetRoomStatus";
```

Comment:
There is no need to start the file name with capital letter, getRoomStatus is just fine

Reply:
that is the name of the file and I had an error on that line before